src/index.js

```diff
@@ -25,27 +25,30 @@ let started = false;
 
 // To be call from the main process
 function setup(webContents, { socketTimeout, socketKeepAliveDelay } = {}) {
-  // Will be called by the renderer process
-  ipcMain.on(START_NOTIFICATION_SERVICE, async (_, senderId) => {
-    // Retrieve saved credentials
+  // Config shape:
```

Comment:
Nit: acknowledging we don't have TS here - could we define the type with inline JSDocs syntax instead?

Reply:
I opted not to because it was fairly verbose with the nested callback but I can take another stab at it. In this situation I don't think the jsdoc gives much benefit since there is no caller

Reply:
Updated 👍🏻